config/xml_schemas/config_batch.xsd

```diff
@@ -57,9 +58,12 @@
         <xs:element minOccurs="0" ref="jobid_pattern"/>
 
   <!-- depend_string: How to indicate to batch system that this job should not run until
-       a previous job has completed successfully -->
+       a previous job has completed (possibly unsuccessfully) -->
```

Comment:
@mfdeakin-sandia There is a problem with changing from completes successfully to completes - if RESUBMIT > 0 and 
the first job fails, in the old method the job would complete on the first failure.   Now it will continue to 
submit jobs until the RESUBMIT flag is decremented to 0.  I don't think that is what you intended can you find a way to run the st_archive but then not RESUBMIT if the first job fails?